README.md

```diff
@@ -33,8 +33,9 @@ First - create initializer, where you can define what services you want to be ch
 Example:
 ```ruby
 StatusCheck.configure do |c|
-  c.check(:postgresql, connection: ->{ ActiveRecord::Base.connection })
-  c.check(:redis,      connection: ->{ Redis.new(url: ENV['REDIS_URL']) })
+  c.check(:my_service, service: StatusCheck::Services::BlockResult, connection: ->{ MyService.is_working? })
+  c.check(:database, service: StatusCheck::Services::ActiveRecordSql, connection: ->{ ActiveRecord::Base.connection })
+  c.check(:redis, service: StatusCheck::Services::ActiveRecordSql, connection: ->{ Redis.new(url: ENV['REDIS_URL']) })
```

Comment:
I don't think it's a good idea to share internal details of 3rd party library to the host application.
Quick example: `ActiberecordSql` was renamed to `Sql::Activerecord` since we've added `rom`. Result - will be unknown number of applications that could not start because of exception in initializer.

Probably it's better to inverse control, and pass symbol or string into `check` method, and than map it somewhere.
Something like
```ruby
c.check(:my_service, service: :block, connection: ->{ MyService.is_working? })
c.check(:database,   service: :activerecord_sql, connection: ->{ ActiveRecord::Base.connection })
```

Reply:
Do you suggest supporting symbol and class definition?

My idea was to give flexibility so anyone can create their own service if needed.